src/utils/search_packages.ts

```diff
@@ -10,7 +10,10 @@ export type PackageResult = {
   maturity: MaturityValue;
 };
 
-export default async function search_packages(query: string) {
+export default async function search_packages(
```

Comment:
the search should reach out to pypi,npm conda forge respectivley to find the package. it should no longer use the db